webpack.config.js

```diff
@@ -0,0 +1,33 @@
+var webpack = require('webpack');
+
+
+module.exports = {
+    entry: {
+        'birthdate': './front/birthdate.js',
+        'housing': './front/housing.js',
+        'lib': './front/lib/index.js',
+    },
+    output: {
+        path: './dist/js',
+        filename: '[name].js',
+    },
+    module: {
+        loaders: [
+            {
+                test: /store\.js$/,
+                loader: 'expose',
```

Comment:
Minimized exposure, still required for the temporary “recap” template. Will be removed once this template is properly coded.